src/views/Jobs.vue

```diff
@@ -247,15 +247,15 @@ export default {
     setFilteredJobs() {
       this.allFilteredJobs = this.jobs.filter(job => {
         // default filter is on current year month
-        const year = this.filter.year[0] || new Date().getFullYear();
-        const month = this.filter.month[0] || new Date().getMonth() + 1;
+        const year = this.filter.year[0];
+        const month = this.filter.month[0];
         const jobStart = new Date(job.starttime);
 
         const match = key => this.matchFilter(job, key);
 
         return (
-          jobStart.getFullYear() === year &&
-          jobStart.getMonth() + 1 === month &&
+          (year ? jobStart.getFullYear() === year : true) &&
```

Comment:
Bit difficult to read, maybe this is easier to read?

`year || jobStart.getFullYear() === year`

Reply:
als year falsy is dan wordt het in jouw voorstel:
false || jobStart.getFullYear() === year => false || false => false
terwijl als year false is in mijn PR wordt het
false ? jobStart.getFullYear() === year : true => true
Dat is niet hetzelfde :-(

Reply:
Yes you''re right there. :)

What about

`!year || jobStart.getFullYear() === year`